app/components/onboarding_email_form/onboarding_email_form.html.erb

```diff
@@ -31,6 +31,18 @@
         <%= c 'checkbox', **field.input_defaults, required: true %>
       <% end %>
 
+      <% if Setting.ask_for_additional_consent && Setting.additional_consent_defined %>
```

Comment:
Can we prefix the setting with `onboarding_` to be consistent with the other onboarding-related settings?